app/subscriber/src/features/my-reports/edit/ReportEditContext.tsx

```diff
@@ -126,6 +135,20 @@ export const ReportEditContextProvider: React.FC<IReportEditContextProviderProps
   const instance = values.instances.length ? values.instances[0] : undefined;
   const [activeInstance, setActiveInstance] = React.useState<IReportInstanceModel>();
   const [previewLastUpdatedOn, setPreviewLastUpdatedOn] = React.useState<string | undefined>();
+  const [showReportData, setShowReportData] = React.useState(
+    values.instances.length > 0 && values.instances[0].content.length > 0,
+  );
+  const [testData, setTestData] = React.useState(
+    generateReportInstanceContents({
```

Comment:
I want to make this configurable by the user later.  Then they can create custom data that more reflects what their report might look like.